packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js

```diff
@@ -631,6 +634,22 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     };
   }
 
+  _invertScrolling(invert) {
+    const scrollNode = this.getScrollableNode();
+    if (invert) {
+      scrollNode.addEventListener("wheel", this._invertedWheelEvent);
+    } else {
+      scrollNode.removeEventListener("wheel", this._invertedWheelEvent);
+    }
+  }
+
+  _invertedWheelEvent = (e) => {
+    const scrollNode = this.getScrollableNode();
```

Comment:
We could avoid another call to get the node by passing it from the first site that gets the node